docs/intro/intro_scaling.rst

```diff
@@ -97,7 +97,7 @@ Time the construction:
 
     bin_packing(weights)
 
-.. code-block:: python
+.. code-block:: text
```

Comment:
Isn't `text` default? I.e. no need to specify it explicitly.

Also, isn't this removing syntax coloring?

Reply:
Yes and yes. But, I think it worth explicitly specifying `text` so anyone in the file in future sees that it's deliberate and for such a small snippet, lack of coloring is no catastrophe:

![image](https://github.com/dwavesystems/dimod/assets/34041130/d0fe3d77-1a85-4a89-8d95-80eb0c7a9f0a)

An alternative solution is to create a custom lexar for ipython. Seems overkill for these two-line snippets.